### DISCUSSION TITLE
Wreck hatch does not rotate correctly/ is rotatable

### DISCUSSION THREAD

**Discussion by MadameWooliest** [FakeFishGames/Barotrauma#13773] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Note: mods were in use but hatch was unmodified.

Wreck hatches have the option to be rotated, and the frames can be rotated, but the doors themselves will stay in the same position.

![Wreck Hatch Error Sub Builder](https://github.com/FakeFishGames/Barotrauma/assets/106273903/40c4536b-7c28-42ed-a4f1-2c60ecf16ab0)

![Wreck Hatch Error Test Mode](https://github.com/FakeFishGames/Barotrauma/assets/106273903/62f6bd30-1a7d-44b6-8d06-c03246168dd4)

### Reproduction steps

1. Enter sub builder
2. Place item "Wreck Hatch"
3. Rotate item "Wreck Hatch"
4. Observe
5. Enter test mode
6. Observe Wreck Hatch

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

Sub Builder, Sub builder testing

### Version

v1.3.0.3

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in our private repo now (commit https://github.com/FakeFishGames/Barotrauma-development/commit/1a6bfcb5efe6b9b5051c685046d24685a0340409).
